packages/api/cms-api/src/blocks/SvgImageBlock.ts

```diff
@@ -39,6 +39,11 @@ class SvgImageBlockData extends BlockData {
             damFile: {
                 ...data,
                 image: {},
+                license: {
```

Comment:
is there an actual use-case for this data in the block json?

Reply:
No, I don't think so, I removed it